src/plugins/iconv/README.md

```diff
@@ -41,13 +41,25 @@ are still properly encoded for anyone.
 
 ## Example
 
-For example `~/.config/iconv.ini` should be `latin1`, but all users
+For example `iconv/iconv.ini` should be `latin1`, but all users
 have `UTF-8` settings:
 
 ```sh
-sudo kdb mount iconv.ini /example/iconv ini iconv from=UTF-8,to=ISO-8859-1
-file ~/.config/iconv.ini             # iconv.ini: ISO-8859 text
-kdb get user/example/iconv/a         # converts ISO-8859 to UTF-8
-kdb set user/example/iconv/a öäß     # converts UTF-8 to ISO-8859
-```
+# Mount the file `iconv/iconv.ini` using the `ini` plugin together with `iconv`
+sudo kdb mount "$PWD/src/plugins/iconv/iconv/iconv.ini" system/tests/iconv ini iconv from=UTF-8,to=ISO-8859-1
+
+# Check the file type
+file -b "$PWD/src/plugins/iconv/iconv/iconv.ini"
```

Comment:
Is this the only occurrence where file is used?

Because this is useless, it only checks the content of a file from the repo.

We should also use the file utility to check for the correct encodings after conversions happened.

Reply:
> Because this is useless,…

I do not think so. I think the command works both as a nice piece of user documentation and it also makes sure, that the test fails

- if someone changes the file encoding of `iconv.ini`, and
- the `iconv` plugin does not change the encoding at all

.

> We should also use the file utility to check for the correct encodings after conversions happened.

I changed the command to check the file type of the mounted file in the latest commit.